typescript/phoenix-sdk/src/market.ts

```diff
@@ -22,24 +23,31 @@ export type OrderId = {
 export type RestingOrder = {
   traderIndex: beet.bignum;
   numBaseLots: beet.bignum;
+  padding: beet.bignum[]; // size: 2
 };
 
 export type TraderState = {
   quoteLotsLocked: beet.bignum;
   quoteLotsFree: beet.bignum;
   baseLotsLocked: beet.bignum;
   baseLotsFree: beet.bignum;
+  padding: beet.bignum[]; // size: 8
 };
 
 export type Ladder = {
   bids: Array<[BN, BN]>;
   asks: Array<[BN, BN]>;
 };
+
 export type UiLadder = {
   bids: Array<[number, number]>;
   asks: Array<[number, number]>;
 };
 
+type PubkeyWrapper = {
```

Comment:
since this is specific to beet rather than the market let's put it in utls/beet.ts